data.md

```diff
@@ -205,21 +169,8 @@ Care is needed for `^Word` to avoid big exponentiation.
     syntax Int ::= Int "^Word" Int [function]
     syntax Int ::= powmod(Int, Int, Int) [function]
  // -----------------------------------------------
-```
-
-```{.k .java}
-    rule W0 ^Word W1 => (W0 ^Word (W1 /Int 2)) ^Word 2  requires W1 >=Int pow16 andBool W1 modInt 2 ==Int 0
-    rule W0 ^Word W1 => (W0 ^Word (W1 -Int 1)) *Word W0 requires W1 >=Int pow16 andBool W1 modInt 2 ==Int 1
-    rule W0 ^Word W1 => (W0 ^Int W1) modInt pow256      requires W1 <Int pow16
-```
-
-RV-K has a more efficient power-modulus operator.
-
-```{.k .ocaml .node}
     rule W0 ^Word W1 => powmod(W0, W1, pow256)
-```
 
-```k
     rule powmod(W0, W1, W2) => W0 ^%Int W1 W2 requires W2 =/=Int 0
```

Comment:
Should we just go straight through `_^%Int__` instead of through `powmod` now? It seems unneeded.